### ISSUE TITLE
Dev: Implement error message updates for Name field (A, AAAA, MX, CNAME, TXT)

### ISSUE THREAD

**Issue by Katherine-MN** [cisagov/getgov#5221]
### Issue description

In our recent design & accessibility review for II Beam (#4727 | [Slack Canvas](https://dhscisa.enterprise.slack.com/docs/T02QH7E1MHA/F097U65KVE1)) we uncovered that there are some inconsistencies in our error messages for the Name field on A, AAAA, MX, CNAME, and TXT records. These are the issues:
* Issue 1:  A Name field accepts an @ symbol when it is the only character used. If it is used with other characters, it is invalid and will throw an error. However, the error message we display in this case is, "Enter the name without using parentheses, colons, or semicolons."  
* Issue 2: The Name field does not accept consecutive dots (...) or leading or trailing dots. However, the error message we display in this case is, "Enter the name without using consecutive periods."

The purpose of this ticket is to implement change to these two messages.

### Acceptance criteria

- [ ] Make changes as shown in the [alert messaging spreadsheet](https://docs.google.com/spreadsheets/d/1hF7_n_zo3EpfbgqqqPUMtSMZXynMtRMsBMlcIqZ91ns/edit?gid=987899751#gid=987899751)
  - Updates are in the "Domain info" tab under "Add/Update DNS records"--relevant rows are marked with this ticket number
- [ ] Contact @montecristo99 when work is complete so she can update the alert spreadsheet.

### Links to other issues

- #5117 

_No response_

**Comment by Katherine-MN** [comment]
@kimallen I was cleaning up the alert spreadsheet and cross-referencing tickets, and noticed this was the only pending implementation ticket not assigned to an epic. Tagging you in case it wasn't on your radar!
